tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/particles.tsx

```diff
@@ -0,0 +1,8 @@
+import { CheckboxInput, FeatureToggle } from '../base';
+
+export const particles: FeatureToggle = {
+  name: 'Render particles',
+  category: 'GAMEPLAY',
+  description: 'Disable to not render most particles, can help with frames.',
```

Comment:
```suggestion
  description: 'Whether or not to render most particles. Disabling can help with getting better framerate.',
```

Reply:
Less negative wording--"disable to not"--and less ambiguous jargon (just "frames").